.circleci/unittest/windows/scripts/set_cuda_envs.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+
+if [ "${CU_VERSION:-}" != "cpu" ] ; then
+    if [[ ${#CU_VERSION} -eq 5 ]]; then
+        CUDA_VERSION="${CU_VERSION:2:2}.${CU_VERSION:4:1}"
+    fi
```

Comment:
Please print an error, if it is not the case
```suggestion
    if [[ ${#CU_VERSION} -eq 5 ]]; then
        CUDA_VERSION="${CU_VERSION:2:2}.${CU_VERSION:4:1}"
    else
        echo "Unrecognized CU_VERSION format:", ${CU_VERSION}
        exit 1
    fi
```

Reply:
@malfet 
The set_cuda_envs.sh is used for unittest_windows_gpu and cmake_windows_gpu.
In unittest_windows_gpu, the passed environment name is CUDA_VERSION (10.2)
https://github.com/pytorch/vision/blob/fe3118c51b487faf1aa59861ffd08b5658f41254/.circleci/config.yml#L742

In cmake_windows_gpu, the passed environment name is cu_version (cu102)
https://github.com/pytorch/vision/blob/fe3118c51b487faf1aa59861ffd08b5658f41254/.circleci/config.yml#L1704

To compatible with these 2 workflows, CU_VERSION(cmake_windows_gpu) is checked first, if no CU_VERSION, I'll use CUDA_VERSION (unittest_windows_gpu).

It looks a little ugly. Maybe we could make the refactory in the future.